### PR TITLE
feat: display router when log app

### PR DESCRIPTION
**Pull request by monkindey**
### Checklist

- [x] `npm test` passes
- ~~tests and/or benchmarks are included~~
- ~~documentation is changed or added~~
- [x] commit message follows commit guidelines

### Affected core subsystem(s)

No

### Description of change

The developer will be less confused  when they run 
```js
console.log(app)
```
`console` will not swallow the `router` property of `app`

More information in https://github.com/eggjs/egg/issues/2228

### CODE REVIEW

File: lib/egg.js [hunk 1]
```diff
@@ -184,10 +184,11 @@ class EggApplication extends EggCore {
     abbr(res, this, [
       'config',
       'controller',
-      'serviceClasses',
-      'middlewares',
       'httpclient',
       'loggers',
+      'middlewares',
+      'router',
+      'serviceClasses',
```
Comment (monkindey) [inline lib/egg.js]: alphabetical

File: lib/egg.js [hunk 2]
```diff
@@ -184,10 +184,11 @@ class EggApplication extends EggCore {
     abbr(res, this, [
       'config',
       'controller',
-      'serviceClasses',
-      'middlewares',
       'httpclient',
       'loggers',
+      'middlewares',
```
Comment (atian25) [inline lib/egg.js]: 这个应该可以去掉 s 了

Comment (monkindey) [reply lib/egg.js]: 需要也修改下吗？

Comment (atian25) [reply lib/egg.js]: 可以看看还有没有其他的补充的，继续 PR

**Pull request merged**